Makefile

```diff
@@ -77,6 +77,10 @@ lint:
 vendor:
 	./hack/vendor.sh
 
+# Alias for vendor (used by verify-deps)
+.PHONY: deps
+deps: vendor
+
```

Comment:
I can't find where this is used

I checked [here](https://github.com/openshift/release/blob/917a988aced215a7ab5eda3709067c0ba3de3dd6/ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml#L267-L272) on the `verify-deps` job config for ccapio, but that [doesn't seem to be using this make target](https://github.com/openshift/release/blob/917a988aced215a7ab5eda3709067c0ba3de3dd6/ci-operator/step-registry/go/verify-deps/go-verify-deps-commands.sh)

Reply:
@theobarberbany I was not implying we should change the `verify-deps` one :)

I was just following the above comment and not undestanding what it meant
> `# Alias for vendor (used by verify-deps)`

If we it that alias is not used by anything and is just an alias I suggest we drop it